lib/src/store.dart

```diff
@@ -153,16 +153,31 @@ class Store extends Stream<Store> with Disposable {
   /// called until that future has resolved.
   ///
   /// If the `Store` has been disposed, this method throws a [StateError].
-  void triggerOnActionV2<T>(Action<T> action,
-      [FutureOr<dynamic> onAction(T payload)]) {
-    if (isOrWillBeDisposed) {
-      throw StateError('Store of type $runtimeType has been disposed');
-    }
+  @deprecated
```

Comment:
Was there a reason this was deprecated and triggerOnAction1 added, other than cleanliness and improved naming consistency? Those seem like good enough reasons, but I was curious if there was some other factor

Reply:
Nope that was it